activesupport/lib/active_support/encrypted_file.rb

```diff
@@ -49,6 +49,13 @@ def key
       read_env_key || read_key_file || handle_missing_key
     end
 
+    # Reads the file and returns the decrypted content.
+    #
+    # Raises:
+    # - MissingContentError if unable to decrypt content.
+    # - MissingKeyError if key is missing and <tt>raise_if_missing_key</tt> is true.
```

Comment:
`MissingContentError` is primarily for when the encrypted file is missing, though it will be raised if the key is missing and a `MissingKeyError` has not already been raised.  Therefore, let's put these in chronological order:

```suggestion
    # - MissingKeyError if the key is missing and +raise_if_missing_key+ is true.
    # - MissingContentError if the encrypted file does not exist or otherwise if the key is missing.
```